src/components/header-block/header-block.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import styled from 'styled-components';
+import { css } from 'styled-components';
+import { Illustration, IllustrationSize } from '../illustration/illustration';
+import { down, up } from '../breakpoint/breakpoint';
+
+const TopLineR = css`
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 110%;
+`;
+
+const HeadlineXL = css`
+  font-weight: bold;
+  font-size: 48px;
+  line-height: 110%;
+`;
+
+const TextS = css`
+  font-size: 14px;
+  line-height: 150%;
+`;
+
+const TextL = css`
+  font-weight: normal;
+  font-size: 20px;
+  line-height: 150%;
+`;
+
+const Text = css`
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 150%;
+`;
```

Comment:
as agreed it's inline for now. thanks!
how did you like using them later in the component. does it feel good?

Reply:
Yes, I do like it. Very clean approach.